### ISSUE TITLE
Double issue in `ICT.bib/Library/Research Articles /by transportomic target/`

### ISSUE THREAD

**Issue by Feat-FeAR** [TCP-Lab/ICT.bib#4]
1. One entry is annotated with `Transporter` instead of `Transporters`, thus creating a new sub-category
2. That entry also has a slash in its title that produces a subdirectory!!

The culprit is:
(2017) - Saccharomyces cerevisiae: [A CRISPR/Cas9-based exploration into the elusive mechanism for lactate export in Saccharomyces cerevisiae](https://doi.org/10.1093/femsyr/fox085)

**Comment by Feat-FeAR** [comment]
I found out that the same also happens for:

- Cancer Reviews
   - 33856653 - Alteration and dysfunction of ion channels/transporters in a hypoxic microenvironment results in the development and progression of gastric cancer
- Non cancer Reviews
   - 35276516 - Expression alteration and dysfunction of ion channels/transporters in the parietal cells induces gastric diffused mucosal injury
   - 15365806 - Genetic disorders of transporters/channels in the inner ear and their relation to the kidney

**Comment by Feat-FeAR** [comment]
A bit more systematic search returned 9 results (only the first 4 are interesting for ICT::bib):

```
$ grep 'title = {' library.bib | grep '/'

title = {{A CRISPR/Cas9-based exploration into the elusive mechanism for lactate export in Saccharomyces cerevisiae}},
title = {{Alteration and dysfunction of ion channels/transporters in a hypoxic microenvironment results in the development and progression of gastric cancer}},
title = {{Genetic disorders of transporters/channels in the inner ear and their relation to the kidney}},
title = {{Expression alteration and dysfunction of ion channels/transporters in the parietal cells induces gastric diffused mucosal injury}},
title = {{Middle-down approach: a choice to sequence and characterize proteins/proteomes by mass spectrometry}},
title = {{THE CONCISE GUIDE TO PHARMACOLOGY 2019/20: Introduction and Other Protein Targets}},
title = {{Accessing Expert-Curated Pharmacological Data in the IUPHAR/BPS Guide to PHARMACOLOGY}},
title = {{The IUPHAR/BPS guide to PHARMACOLOGY in 2022: curating pharmacology for COVID-19, malaria and antibacterials}},
title = {{The mRNA transportome of the BicD/Egl transport machinery}},
```

**Comment by Feat-FeAR** [comment]
Modifying titles in the bib file does not seem the good way to go... We should implement a check in the python script to prevent filenames having slashes (maybe sobstitute them with an hypen).

**Comment by Feat-FeAR** [comment]
Solved by commit 5bbc3dee4a6d5529ab237ba80abcfbbf74984651
